src/connector/mssql.rs

```diff
@@ -326,6 +330,7 @@ impl Queryable for Mssql {
         .await
     }
 
+    #[tracing::instrument(skip(self))]
```

Comment:
same as previous comment

Reply:
And same answer too. I try to skip as much params as I can from instrumentation, but the SQL query is something we must have there for our users in the future. Think of this whole thing as a product in the Prisma Studio :)